manifests/r/ransome1/sleek/1.2.0/ransome1.sleek.installer.yaml

```diff
@@ -13,9 +11,15 @@ Installers:
   InstallerUrl: https://github.com/ransome1/sleek/releases/download/v1.2.0/sleek-1.2.0-win-Setup.exe
   InstallerSha256: 81E967109DA9BEEC0A8162D569C4B0E961B0C3F2309F5DEC4C3C4AC3FC921F39
   UpgradeBehavior: install
+  AppsAndFeaturesEntries:
+  - DisplayName: sleek 1.2.0
+    ProductCode: 5b6e5ce6-6f74-5ccb-a937-b1c9ab815902
 - Architecture: x64
   InstallerType: portable
   InstallerUrl: https://github.com/ransome1/sleek/releases/download/v1.2.0/sleek-1.2.0-win.exe
   InstallerSha256: AD7CED8FC2598B5BCC08B992861950F8348D996570E1690E6691E61EFF03619B
+  AppsAndFeaturesEntries:
+  - ProductCode: ransome1.sleek__DefaultSource
```

Comment:
This is a automatically generated product code from winget. I need to put a filter on the script to remove these (or skip portable app types), since the ARP entries are managed by winget.

Can you remove this?
```suggestion
```

Reply:
BTW, I just added a filter to the script. It should be safe to run against manifests with portable installer manifests.

Reply:
Have you committed the change?